pom.xml

```diff
@@ -187,6 +187,7 @@
 		<de.flapdoodle.embed.mongo.version>2.2.0</de.flapdoodle.embed.mongo.version>
 		<hamcrest.version>2.2</hamcrest.version>
 		<org.xmlunit.version>2.8.3</org.xmlunit.version>
+		<tomcat.version>9.0.56</tomcat.version>
```

Comment:
We're on .59 in Gradle/downloaded bundle. Also, why do we need this for Event Bus?

Reply:
I'll update the version, it's only required to be present in the classpath for tests it's not included in the final war